process/mock/requestedItemsHandlerMock.go

```diff
@@ -0,0 +1,18 @@
+package mock
+
+type RequestedItemsHandlerMock struct {
```

Comment:
This is a Stub, might rename the struct and also the file.

Reply:
Done